### PR TITLE
[MAINTENANCE] Change all instances of `create_expectation_suite` to `add_expectation_suite` in tests, docs, and source code

### PR DESCRIPTION
**Pull request by cdkini**
Changes proposed in this pull request:
- Remove deprecated code calls

### Definition of Done
- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added [unit tests](https://docs.greatexpectations.io/docs/contributing/contributing_test#writing-unit-and-integration-tests) where applicable and made sure that new and existing tests are passing.
- [x] I have run any local integration tests and made sure that nothing is broken.

### CODE REVIEW

**Comment by ghost** [comment]
<details open>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=7117&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

#### 

[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=7117&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
CodeSee Map Legend
</details>

File: great_expectations/validator/validator.py [hunk 1]
```diff
@@ -113,7 +85,33 @@
     )
 
 if TYPE_CHECKING:
+    from great_expectations.core.batch import (
```
Comment (NathanFarmer) [inline great_expectations/validator/validator.py:92]: ❤️ 🙇🏻

File: tests/integration/docusaurus/connecting_to_your_data/database/athena_python_example.py [hunk 2]
```diff
@@ -74,7 +74,7 @@
         f'Loaded ExpectationSuite "{suite.expectation_suite_name}" containing {len(suite.expectations)} expectations.'
     )
 except DataContextError:
-    suite = context.create_expectation_suite(
```
Comment (NathanFarmer) [inline tests/integration/docusaurus/connecting_to_your_data/database/athena_python_example.py:77]: Shouldn't this just be `add_expectation_suite`?

Comment (NathanFarmer) [reply tests/integration/docusaurus/connecting_to_your_data/database/athena_python_example.py:77]: Didn't we confirm that `get` won't work above?

**Review by Kilo59 — APPROVED** [review]
Just want to confirm that the old method still exists and is being deprecated in this PR?
Most of these changes are just changing us to use the new `add_expectation_suite` internally?

File: tests/integration/docusaurus/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_pandas_dataframe.py [hunk 3]
```diff
@@ -16,7 +16,7 @@
 
 # create and load Expectation Suite
 # <snippet name="tests/integration/docusaurus/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_pandas_dataframe.py create_expectation_suite">
-context.create_expectation_suite(
```
Comment (NathanFarmer) [inline tests/integration/docusaurus/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_pandas_dataframe.py:19]: `add_expectation_suite`?

File: tests/integration/docusaurus/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_dataframe.py [hunk 4]
```diff
@@ -21,7 +21,7 @@
 
 # create and load Expectation Suite
 # <snippet name="tests/integration/docusaurus/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_dataframe.py create_expectation_suite">
-context.create_expectation_suite(
```
Comment (NathanFarmer) [inline tests/integration/docusaurus/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_dataframe.py:24]: `add_expectation_suite`?

File: tests/integration/docusaurus/deployment_patterns/aws_glue_deployment_patterns.py [hunk 5]
```diff
@@ -43,7 +43,7 @@
 context_gx = get_context(project_config=config)
 
 expectation_suite_name = "suite_name"
-suite = context_gx.create_expectation_suite(expectation_suite_name)
```
Comment (NathanFarmer) [inline tests/integration/docusaurus/deployment_patterns/aws_glue_deployment_patterns.py:54]: `add_expectation_suite`?

**Review by NathanFarmer — APPROVED** [review]
A few questionable ones before merging

**Pull request merged**